lib/src/report/multi_platform.dart

```diff
@@ -117,11 +117,47 @@ Future<ReportSection> multiPlatform(String packageDir, Pubspec pubspec) async {
     );
   }
 
+  final wasmSubsection = await _createWasmSubsection(context);
+
   return makeSection(
       id: ReportSectionId.platform,
       title: 'Platform support',
       maxPoints: 20,
-      basePath: packageDir,
-      subsections: [subsection],
+      basePath: context.packageDir,
+      subsections: [subsection, wasmSubsection],
       maxIssues: 20);
 }
+
+Future<Subsection> _createWasmSubsection(PackageContext context) async {
+  final tr = await context.staticAnalysis;
+  final description = 'WASM readiness';
+  final explanation =
+      tr.explanations.where((e) => e.tag == PanaTags.isWasmReady).firstOrNull;
+  if (explanation != null) {
+    return Subsection(
+      description,
+      [explanationToIssue(explanation)],
+      0,
+      0,
+      ReportStatus.failed,
+    );
+  }
+
+  if (tr.tags.contains(PanaTags.isWasmReady)) {
+    return Subsection(
+      description,
+      [RawParagraph('Package is compatible with runtime `wasm`.')],
```

Comment:
Do we have somewhere to link to?
Perhaps:
```suggestion
      [RawParagraph('Package is compatible with runtime `wasm`. See <https://dart.dev/web/wasm> for details.')],
```